shell/src/shared/atoms/UserImage.svelte

```diff
@@ -12,12 +12,16 @@ export let transparent: boolean = false;
 export let tooltip: boolean = false;
 export let profileLink: boolean = true;
 export let editable: boolean = false;
+export let image: string = "";
 let displayName: string = "";
 let isOrganisation: boolean = false;
 let sizeInPixels = 0;
+let noAvatar;
 
-const seed = Web3.utils.hexToNumber(profile.circlesAddress.slice(0, 15));
-let noAvatar = jazzicon(size === 15 ? 54 : size * 4, seed);
+if (!image) {
```

Comment:
isn't this also 'true' for all the calls for ProfileImage where just the profile is attached?
so in those cases we could skip the overhead of the web3 call and generating of the jazzicon, no?
so maybe do if(!image && !profile.avatarUrl) instead?

Reply:
done. please check again @codeho